cypress/integration/ErrorHandling/MissingAction.spec.js

```diff
@@ -10,44 +10,66 @@ import * as editDialogModal from '../../support/components/EditDialogModal'
 import * as common from '../../support/Common'
 import * as actions from '../../support/Actions'
 import * as scorerModal from '../../support/components/ScorerModal'
+import * as helpers from '../../support/Helpers'
 
-describe('ErrorHandling', () => {
-  it('Missing Action', () => {
-    models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
-    modelPage.NavigateToTrainDialogs()
-    cy.WaitForTrainingStatusCompleted()
-
-    modelPage.VerifyNoErrorIconOnPage()
+describe('Missing Action - ErrorHandling', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Setup', () => {
+    it('Should import a model and wait for training to complete', () => {
+      models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
+      modelPage.NavigateToTrainDialogs()
+      cy.WaitForTrainingStatusCompleted()
+    })
 
+    it('Should verify there are no error icons on the page and create a new Train Dialog', () => {
+      modelPage.VerifyNoErrorIconOnPage()
     train.CreateNewTrainDialog()
+    })
+  })
 
-    train.TypeYourMessage(common.gonnaDeleteAnAction)
-    editDialogModal.ClickScoreActionsButton()
-    train.SelectAction(common.whatsYourName)
-    
-    train.Save()
-
-    modelPage.NavigateToActions()
-    actions.DeleteAction(common.whatsYourName)
-    modelPage.NavigateToTrainDialogs()
-
-    modelPage.VerifyErrorIconForTrainDialogs()
-    train.VerifyErrorsFoundInTraining(`${common.validityIcon}${common.gonnaDeleteAnAction}`, common.gonnaDeleteAnAction, undefined)
-
-    train.EditTraining(`${common.validityIcon}${common.gonnaDeleteAnAction}`, common.gonnaDeleteAnAction, undefined)
+  context('Train', () => {
+    it('Should complete and save a simple 1 action Train Dialog', () => {
+      train.TypeYourMessage(common.gonnaDeleteAnAction)
+      editDialogModal.ClickScoreActionsButton()
+      train.SelectAction(common.whatsYourName)
+      train.Save()
+    })
+  })
 
-    editDialogModal.VerifyErrorMessage(trainDialogHasErrorsMessage)
+  context('Action', () => {
+    it('Should delete the action we just used in the Train Dialog', () => {
+      modelPage.NavigateToActions()
+      actions.DeleteAction(common.whatsYourName)
+    })
+  })
 
-    editDialogModal.SelectChatTurnStartsWith('ERROR: Can’t find Action Id')
-    editDialogModal.VerifyErrorMessage('Action does not exist')
-    scorerModal.VerifyMissingActionNotice()
+  context('Train Dialog Grid', () => {
```

Comment:
Would prefer this to be in plural form "Train Dialogs" can add "Grid" if you want to make it clear it's the list view.  There are other lists/grids for the memory and action selection on the train dialog so this is confusing to me.

One is verifying the error icon in the list view and the other is verify the more specific error content in the modal.

Reply:
Made it plural. 

I am not sure I understood the rest of your comment, but I'd be happy to reword anything else so as to keep the language consistent with whatever way the team refers to each component in question.